prboom2/src/gl_sky.c

```diff
@@ -437,7 +437,7 @@ void gld_DrawScreenSkybox(void)
   {
     #define WRAPANGLE (ANGLE_MAX/4)
 
-    float fU1, fU2, fV1, fV2;
+    double fU1, fU2, fV1, fV2;
```

Comment:
These variables are consumed as floats by the gl functions below, right? Won't they just lose the precision anyway?

Reply:
It's less loss overall to round at the end when calling gl than to use float for each intermediate result.

Reply:
👍